lib/node_modules/@stdlib/repl/lib/ansi_colors.js

```diff
@@ -0,0 +1,58 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MAIN //
+
+/**
+* Table mapping generic color names to their ANSI color codes.
+*
+* @private
+* @name ANSI
+* @type {Object}
+*/
+var ANSI = {
+	// Original colors:
+	'black': '\u001b[30m',
```

Comment:
Let's get this in for now as-is, but as a follow-up it may be worthwhile to add a separate package specifically for colors, similar to https://www.npmjs.com/package/colors or https://www.npmjs.com/package/chalk. Similar to what `chalk` is doing, we could do feature detection in there for which ANSI colors are supported and adjust the exports accordingly. A potential location for such a package may be `@stdlib/cli/colors`, with color support potentially living in the `assert` namespace.

Reply:
That would be nice to have! will add it as a TODO.